src/ansys/rocky/core/client.py

```diff
@@ -23,18 +23,19 @@
 Module that defines the ``RockyClient`` class, which acts as a proxy for a Rocky
 application session.
 """
-from collections.abc import Callable, Generator
-import pickle
-from typing import Any, Final
+import os
+from typing import Final
 
 import Pyro5.api
-import serpent
 
-from ansys.rocky.core.exceptions import RockyApiError
+from ansys.rocky.core.serializers import register_proxies
+
+register_proxies()
```

Comment:
Calling functions at import time should be the last resort. Do we have any alternative?